stl/src/excptptr.cpp

```diff
@@ -67,7 +67,7 @@ namespace {
     [[nodiscard]] _Ty& _Immortalize() noexcept {
         return _Immortalize_impl<_Ty>._Storage;
     }
-#else // choose immortalize strategy
+#else // ^^^ !defined(_M_CEE) / defined(_M_CEE), TRANSITION, VSO-1153256 vvv

```

Comment:
With the TRANSITION here and on line 89, I'm inclined to remove the comment on 46 so the comments bracket the section of code that will actually go away when VSO-1153256 is fixed.

Reply:
Good idea!